modules/ob-deleting-the-resources.adoc

```diff
@@ -0,0 +1,5 @@
+:_mod-docs-content-type: CONCEPT
+[id="ob-deleting-the-resources_{context}"]
+= Deleting the resources
+
+You can delete the resources that are created by `buildah`, `source-to-image` and `buildpacks` build processes.
```

Comment:
how to delete these resources? Kindly explain it at the start as the sentence looks incomplete.